Client/src/Pages/Integrations/index.jsx

```diff
@@ -1,9 +1,40 @@
-import React from 'react'
+import IntegrationsComponent from '../../Components/Integrations';
+import { Box, Typography, useTheme } from '@mui/material';
 
+/**
+ * Integrations Page Component
+ * @returns {JSX.Element}
+ */
 const Integrations = () => {
+    const theme = useTheme();
+  
     return (
-        <div>Integrations</div>
-    )
-}
-
-export default Integrations
\ No newline at end of file
+      <Box 
+        display="flex"
+        flexDirection="column"
+        alignItems="flex-start"
+        justifyContent="flex-start"
+        height="100vh"
+        p={theme.spacing(4)}
+        mt={theme.spacing(8)} 
+      >
+        <Typography variant="h4" component="h1" gutterBottom>
+          Integrations
+        </Typography>
+        <Typography variant="h6" component="h2" gutterBottom>
+          Connect Uptime Genie to your favorite service
+        </Typography>
```

Comment:
Can you also make sure this one has 13px font-size?

Reply:
Addressed!